styles/styles.css

```diff
@@ -337,6 +337,10 @@ main .caption {
   font-size: var(--body-font-size-xs);
   color: var(--color-gray-700);
 } 
+main .figure .caption {
+  padding: 0;
```

Comment:
is this addition needed or just a precaution?
i would rather avoid that unless we think it is required... @fkakatie

Reply:
I think the idea here is to negate the padding applied to all `<p>` tags, as that's where `.caption` is applied.